Rakefile

```diff
@@ -222,8 +241,8 @@ namespace :release do
   end
 
   desc 'Updates metadata for the new release'
-  task :update_metadata do
-    print "New version of Sourcery in sematic format [x.y.z]? "
+  task :update_metadata => [:git_tag] do
```

Comment:
I think it's a leftover, sorry I was in a hurry while doing changes, these both comments a good catch of my mistakes 👍 
Check dce5d12 for both updates.